R/dist.R

```diff
@@ -192,21 +193,23 @@ dist_skel <- function(n, dist = FALSE, cum = TRUE, model,
 #' )
 #'
 #' # integer adjusted lognormal model
-#' dist_fit(rlnorm(1:100, log(5), 0.2),
-#'   samples = 1000, dist = "lognormal",
-#'   cores = ifelse(interactive(), 4, 1), verbose = TRUE
-#' )
+# dist_fit(rlnorm(1:100, log(5), 0.2),
+#   samples = 1000, dist = "lognormal",
+#   cores = ifelse(interactive(), 4, 1), verbose = TRUE
+# )
 #' }
-dist_fit <- function(values = NULL, samples = NULL, cores = 1,
+dist_fit <- function(values = NULL, samples = 1000, cores = 1,
```

Comment:
These are all nice changes @jamesmbaazam but if possible can you address in separate issue/PR combos vs here as otherwise this PR may be hard to review. ty ❤️

Reply:
Agreed. I'll do that.